daprdocs/content/en/reference/cli/dapr-mtls-expiry.md

```diff
@@ -0,0 +1,28 @@
+---
+type: docs
+title: "mtls expiry CLI command reference"
+linkTitle: "mtls expiry"
+description: "Detailed information on the mtls expiry CLI command"
```

Comment:
This is correct in that this is not K8s specific. See previous comment on mtls export.

Reply:
no this is kubernetes specific. see the linked go file in github above.